src/webviews/filters/index.js

```diff
@@ -28,6 +29,18 @@ module.exports = class FiltersUI {
         return;
       }
 
+      if (copy) {
+        name = ``;
+        filter = {
+          name: `Filter ${objectFilters.length + 1}`,
+          library: filter.library,
+          object: filter.object,
+          types: [...filter.types],
+          member: filter.member,
+        }
```

Comment:
Instead of naming the copy `Filter XX` it would be nice just to append ` - copy` to the name of the filter being copied:

```suggestion
        filter = {
          name: `${name} - copy`,
          library: filter.library,
          object: filter.object,
          types: [...filter.types],
          member: filter.member,
        }
        name = ``;
```